presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestEnvironments.java

```diff
@@ -31,53 +29,37 @@
 import static com.facebook.presto.execution.resourceGroups.db.H2TestUtil.getDao;
 import static com.facebook.presto.execution.resourceGroups.db.H2TestUtil.getDbConfigUrl;
 
+@Test(singleThreaded = true)
 public class TestEnvironments
 {
     private static final String LONG_LASTING_QUERY = "SELECT COUNT(*) FROM lineitem";
-    private DistributedQueryRunner runner;
 
-    @BeforeGroups(value = TEST_ENVIRONMENT)
-    public void setUpRunnerForEnvironment1()
+    @Test(timeOut = 240_000, groups = TEST_ENVIRONMENT)
```

Comment:
Remove the groups. These are for conditionally running tests and aren't used here at all.

Reply:
yep, this is leftover.